src/components/common/input/FloatingSelect.tsx

```
@@ -68,9 +70,10 @@ const FloatingSelect: React.FC<FloatingSelectProps> = ({
 				height="60px"
 				mt={4}
 			>
+				<option value="" hidden></option>
 				{options.map((option) => (
 					<option key={option.value} value={option.value}>
-						{option.label}
+						{option.label[language] ? option.label[language] : option.label}
```

ESLint

[error] Replace `·?·option.label[language]·` with `⏎↹↹↹↹↹↹↹?·option.label[language]⏎↹↹↹↹↹↹↹`

(prettier/prettier)